flink-core/src/main/java/org/apache/flink/configuration/SecurityOptions.java

```diff
@@ -132,6 +132,22 @@ public class SecurityOptions {
                     .withDescription(
                             "The time period when keytab login happens automatically in order to always have a valid TGT.");
 
+    @Documentation.Section(Documentation.Sections.SECURITY_AUTH_KERBEROS)
+    public static final ConfigOption<Duration> KERBEROS_TOKENS_RETRY_WAIT =
+            key("security.kerberos.tokens.retry-wait")
+                    .durationType()
+                    .defaultValue(Duration.ofHours(1))
+                    .withDescription(
+                            "The time period how long to wait before retrying to obtain new delegation tokens after a failure.");
+
+    @Documentation.Section(Documentation.Sections.SECURITY_AUTH_KERBEROS)
+    public static final ConfigOption<Double> KERBEROS_TOKENS_RENEWAL_RATIO =
+            key("security.kerberos.tokens.renewal-ratio")
```

Comment:
A clearer name would be `security.kerberos.tokens.renewal.time-ratio`

Reply:
Renamed.